_revisions/r1.3.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Version 1.3
+date: 2019-06-04
+category: proposal
+tagline: 
+---
+{% include JB/setup %}
+ - **Download** [pdf](https://raw.githubusercontent.com/openrisc/doc/master/openrisc-arch-1.3-rev1.pdf)
+ - **Changes**
+   - ORFPX64A32 double-precision floating point operations on 32-bit hardware using register pairs (P14)
```

Comment:
Perhaps, one more thing could be added into `Changes`. Something like:
`- CPUCFGR[15] is assigned for ORFPX64A32 presence flag`

Reply:
Sure I have made the update.

Reply:
LGTM. No more wishes from my side.